ksrates/plot_paralogs.py

```diff
@@ -22,7 +22,7 @@ def plot_paralogs_distr(config_file, correction_table_file, paralog_tsv_file, an
     paranome_analysis = config.get_paranome()
     colinearity_analysis = config.get_colinearity()
     if not colinearity_analysis and not paranome_analysis:
-        logging.error("At least 'paranome_analysis' or 'colinearity_analysis' parameters in configuration file needs to be set to 'yes'.")
+        logging.error("Neither paranome nor collinearity was requested in the configuration file. Please select at least one option.")
```

Comment:
```suggestion
        logging.error('At least one of the "paranome" or "collinearity" parameters in the configuration file needs to be set to "yes".')
```

Reply:
I liked the previous style a bit better, more specific and in line with other error messages in the Python code.

Reply:
Alright!